src/CMakeLists.txt

```diff
@@ -62,12 +63,13 @@ if (MSVC)
 		LINK_FLAGS "/subsystem:windows /ENTRY:mainCRTStartup")
 elseif(APPLE)
 	# Help CPack build the bundle
+	set_source_files_properties(${ICON} PROPERTIES
+		MACOSX_PACKAGE_LOCATION "Resources")
 	set_target_properties(${PROJECT_NAME} PROPERTIES
-		RESOURCE glfw.icns
```

Comment:
What happens if you set `RESOURCE ${ICON}` instead of using `set_source_files_properties`?